pkg/groups/groups.go

```diff
@@ -192,3 +192,29 @@ func ResetOffsets(
 		},
 	)
 }
+
+// getOffset gets offset for a given topic partition for resetting offsets for consumer group
+func GetOffset(
+	ctx context.Context,
+	connector *admin.Connector,
+	topic string,
+	strategy string,
+	partition int,
+	offset int64,
```

Comment:
removed offset param from this function, used an if condition which checks if a strategy is specified before calling this function, otherwise uses offset provided.